### PR TITLE
Fix tabbing in widgets not triggering auto-scrolling

### LINKED ISSUE #25927: Widget Editor: Tabbing through widget areas does not scroll the page
**Describe the bug**
When tabbing between widget areas the page scroll position does not follow keyboard focus, and so it's hard to tell where my focus is:

**What was expected**
I expected to see the page scroll follow my keyboard focus.

### PR DESCRIPTION
**Pull request by kevin940726**
## Description

Fix #25927.

It took me way more time than expected. It turns out, we just need to wrap the popover with an additional `position: relative` wrapper... 😞

We probably would want to add an e2e test in the future to prevent regression.

## How has this been tested?

1. Go to the widgets screen
2. Open some widget areas until there's a scrollbar
3. Press <kbd>Tab</kbd> several times to tab through the widgets
4. Observe that when tabbing to an element below the viewport, the screen auto scrolls to it

## Screenshots 
Kapture 2020-10-15 at 13 18 44

## Types of changes

Bug fix

## Checklist:
- [ ] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/edit-widgets/src/components/widget-areas-block-editor-content/index.js [hunk 1]
```diff
@@ -17,31 +17,27 @@ import Notices from '../notices';
...
-						event.preventDefault();
-					} }
-				>
-					<WritingFlow>
-						<ObserveTyping>
-							<BlockList className="edit-widgets-main-block-list" />
-						</ObserveTyping>
-					</WritingFlow>
-				</div>
+		<BlockSelectionClearer>
```
Comment (kevin940726) [inline packages/edit-widgets/src/components/widget-areas-block-editor-content/index.js:21]: I also replaced the original `<div>` with `<BlockSelectionClearer>`. Not sure why we were not using it in the first place?

Comment (talldan) [reply packages/edit-widgets/src/components/widget-areas-block-editor-content/index.js:21]: Probably just something missed at some point.

File: packages/edit-widgets/src/components/widget-areas-block-editor-content/style.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.editor-styles-wrapper {
```
Comment (talldan) [inline packages/edit-widgets/src/components/widget-areas-block-editor-content/style.scss:1]: Minor, but while the `.editor-styles-wrapper` class seems to be needed to pull in particular styles, this classname reuse seems like a bad thing that goes against the BEM-like principles in the project:
https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

A trade-off could be to add a second classname:
```html
<div
    className="editor-styles-wrapper edit-widgets-styles-wrapper"
```

and targeting that instead:
```css
.edit-widgets-styles-wrapper {
    position: relative;
}
```
```

Comment (kevin940726) [reply packages/edit-widgets/src/components/widget-areas-block-editor-content/style.scss]: Good call! Added in 247498e.

**Review by talldan — APPROVED** [review]
LGTM! Great work tracking this one down.

**Pull request merged**